### ISSUE TITLE
Not able to access data from certain Tickers

### ISSUE THREAD

**Issue by martimfj** [ranaroussi/yfinance#112]
I tried to get historical data from Brazilian market, but got some anomalies:

```
import yfinance as yf

for i in ["KLBN11", "SANB11", "YDUQ3"]:
    ticker = yf.Ticker(f"{i}.SA")
    data = ticker.history(period="6mo")["Close"].to_frame()
    data.columns = [i]
    print(data)
```

From KLBN11 and SAMB11 I got only the Close value of the day. YDUQ3 I got 42 days of data. I checked the Yahoo Finance website and was able to see all the data history,

**Comment by ranaroussi** [comment]
Can additional information found on Yahoo!?

**Comment by jasonxu1991** [comment]
Hi im having the same issue like you. For example i can't fetch the information on tickers like "AABA" "NIO" just to name two.

**Comment by ranaroussi** [comment]
Fixed in `0.1.50`. Please re-open if the problem persists.

**Comment by RudolfLapie** [comment]
Using yfinance 0.1.55 and python 3.8.1
Had the same issue again with ticker SGLN.L. 
Ticker.info raises an exception which Ticker.history returns resulst
